OverlayPlugin.Core/Controls/ControlPanel.resx

```diff
@@ -147,7 +147,7 @@
     <value>3, 4, 3, 4</value>
   </data>
   <data name="tabControl.Size" type="System.Drawing.Size, System.Drawing">
-    <value>671, 282</value>
+    <value>671, 270</value>
```

Comment:
Here (and all over in this file, and in other `.resx` files) are changes to size, location, padding, and margin of controls. These changes will likely cause display issues in non-English languages.

I'm guessing the margin/padding are intentional to match styling, but making the elements smaller will cause other languages to be cut off in display.

Reply:
When making the new controls, I did not sufficiently reverse engineer how normal WinForms controls worked.  So when my controls were released, there were certain idiosyncrasies.  One such thing that I was not immediately aware of is that a lot of WinForms controls ignore the Margin property unless they are docked to another control.  My controls however always listened to Margin/Padding and it affected their rendering.  So the majority of Margin changes would not cause any visual effects for normal WinForms controls, but make my controls the approximate visual size of the old controls.  Typically, my controls with 1px margin will look similar to an undocked WinForms control with any value.  I did not look at any Microsoft code when making my controls, so the way my controls handle padding sometimes made visual inspection and fixing necessary.  --  The next version of ACT will have my controls also ignore Margin in the same cases that WinForms controls do... at which time the changed Margin properties will no longer have a meaning.

To my recollection, I don't remember manually changing the size of any controls and I certainly did not explicitly change any .resx files since they are auto-generated by Visual Studio.  I'm not really familiar with control properties being stored in .resx files instead of designer.cs files... so I completely ignored .resx changes with the mindset they were only for localization and had no other runtime effect.  In your example, the size change is meaningless since the control in question is docked to "Fill" its parent control.  The size is just there to help initial layout before docking happens.  One thing I've noticed about user-drawn TabControls is that as soon as you enable it, WinForms gives you something completely different than what you had before.  The default font and tab height is different and this affects reported sizes and how it auto-calculates reserved tab space and TabPage space.  The designer in Visual Studio does not faithfully recreate what it will really look like.  Just opening this kind of thing in Visual Studio can alter the "code" but be meaningless during runtime.  When working with ACT I feel like every time I change one small thing in the designer, 10 other things change... but at runtime I can't see anything different.